sfputil/main.py

```diff
@@ -273,6 +273,18 @@ def convert_sfp_info_to_output_string(sfp_info_dict):
                         output += '{}{}: {}\n'.format((indent * 2), compliance_key, spec_compliance_dict[compliance_key])
                 except ValueError as e:
                     output += '{}N/A\n'.format((indent * 2))
+        elif key == 'application_advertisement':
+            try:
```

Comment:
if the CLI output has been updated, I believe you need to update the unit test accordingly.

Reply:
UT is available he
[CMIS_UT_IX9.log](https://github.com/Azure/sonic-utilities/files/7293691/CMIS_UT_IX9.log)
re